www/src/components/landingPage/banner.astro

```diff
@@ -1,124 +1,121 @@
+                  Github
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    viewBox="0 0 448 512"
+                    class="h-3 ml-2 fill-white"
                   >
-                    <code id="command-text">npx create-t3-app@latest</code>
-                    <svg
-                      id="copy-icon"
-                      xmlns="http://www.w3.org/2000/svg"
-                      width="24"
-                      height="24"
-                      viewBox="0 0 24 24"
-                      fill="none"
-                      stroke="currentColor"
-                      stroke-width="2"
-                      stroke-linecap="round"
-                      stroke-linejoin="round"
-                      ><rect x="9" y="9" width="13" height="13" rx="2" ry="2"
-                      ></rect><path
-                        d="M5 15H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h9a2 2 0 0 1 2 2v1"
-                      ></path>
-                    </svg>
-                    <svg
-                      id="check-icon"
-                      class="hidden"
-                      xmlns="http://www.w3.org/2000/svg"
-                      width="24"
-                      height="24"
-                      viewBox="0 0 24 24"
-                      fill="none"
-                      stroke="currentColor"
-                      stroke-width="2"
-                      stroke-linecap="round"
-                      stroke-linejoin="round"
-                      ><polyline points="20 6 9 17 4 12"></polyline>
-                    </svg>
-                  </button>
-                </div>
+                    <path
+                      d="M256 64C256 46.33 270.3 32 288 32H415.1C415.1 32 415.1 32 415.1 32C420.3 32 424.5 32.86 428.2 34.43C431.1 35.98 435.5 38.27 438.6 41.3C438.6 41.35 438.6 41.4 438.7 41.44C444.9 47.66 447.1 55.78 448 63.9C448 63.94 448 63.97 448 64V192C448 209.7 433.7 224 416 224C398.3 224 384 209.7 384 192V141.3L214.6 310.6C202.1 323.1 181.9 323.1 169.4 310.6C156.9 298.1 156.9 277.9 169.4 265.4L338.7 96H288C270.3 96 256 81.67 256 64V64zM0 128C0 92.65 28.65 64 64 64H160C177.7 64 192 78.33 192 96C192 113.7 177.7 128 160 128H64V416H352V320C352 302.3 366.3 288 384 288C401.7 288 416 302.3 416 320V416C416 451.3 387.3 480 352 480H64C28.65 480 0 451.3 0 416V128z"
+                    ></path>
+                  </svg>
+                </Button>
+              </div>
+              <div class="flex justify-center mt-4 xl:mt-8 h-full relative">
+                <button
+                  class="px-2 py-2 text-lg sm:px-5 sm:py-4 sm:text-xl rounded-md cursor-pointer bg-white/10 hover:bg-white/20 transition-all duration-300 relative flex flex-row items-center gap-2 border border-t3-purple-200/20 hover:border-t3-purple-300/50"
+                  title="Copy the command to get started"
+                  id="command"
+                >
+                  <code id="command-text">npx create-t3-app@latest</code>
```

Comment:
This grows at a different breakpoint than the buttons do, resulting in some weird proportions at some screen sizes.

<img width="733" alt="image" src="https://user-images.githubusercontent.com/8353666/193416227-ed65d361-40aa-4d8f-be44-d35defa93414.png">

I also think it could be a bit smaller than it currently is at _every_ break point.